### ISSUE TITLE
Predicted boiling point is not accurate

### ISSUE THREAD

**Issue by thuongmhh** [marco-hoffmann/GRAPPA#1]
Thank you for sharing the paper, code, and model. When I tested it on several molecules, I noticed that the predicted boiling points differed significantly from their experimental values, indicating that there may still be room for improvement in the model.

Do you think adding experimental boiling points or reported Antoine coefficients to the training set would help increase the model’s accuracy?

**Comment by marco-hoffmann** [comment]
Hi, first of all thank you for your feedback and for using GRAPPA! Could you maybe give me a few examples of the molecules for which you obtained the deviations from experimental data? And could you quantify the deviations? Generally, we found the boiling point prediction to be very accurate (with a mean absolute error of about 5K) on our test set of unseen components.

Concerning your question:
Adding more experimental boiling points to the training data might help. However, it could also lead to the model performance decreasing in other areas (i.e., for temperatures far from the boiling point). While the training to Antoine coefficients is theoretically possible, I personally don't think it would lead to large improvements. Also, this would only allow training to those components, for which sufficient data points are available.

**Comment by thuongmhh** [comment]
Dear Marco, my apologies — I forgot to subtract the pressure when calculating the boiling point.

**Comment by marco-hoffmann** [comment]
No problem!
